internal/option/config.go

```diff
@@ -208,6 +208,16 @@ type CertGenConfig struct {
 	ClustermeshApiserverRemoteCertSecretName string
 }
 
+// getStringWithFallback returns the value associated with the key as a string
+// if it was explicitly set. If key was not explicitly set, returns the value
```

Comment:
I am unsure what "explicitly set" means in this context. The documentation for viper's `IsSet` states:
> IsSet checks to see if the key has been set in any of the data locations. IsSet is case-insensitive for a key.

So a default value also means that the key is set even though the caller did not "explicitely set" the key.